src/shogun/modelselection/GridSearchModelSelection.cpp

```diff
@@ -68,9 +68,6 @@ CParameterCombination* CGridSearchModelSelection::select_model(bool print_state)
 			current_combination->print_tree();
 		}
 
-		current_combination->apply_to_modsel_parameter(
```

Comment:
this is the big fish to replace

Reply:
Could you, instead of removing the method call, just keep it and place a runtime error (SG_NOTIMPLEMENTED) inside the method?